F/FuzzifiED/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FuzzifiED"
+version = v"0.5.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mankai-chow/FuzzifiED_Fortran.git", "c96719aa5fb0e072612de78a1ebe2cf140515d47")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd FuzzifiED_Fortran/
+export MKL_THREADING_LAYER=GNU
+gfortran -fPIC -larpack -fopenmp -c ./cfs.f90
+gfortran -fPIC -larpack -fopenmp -c ./bs.f90
+gfortran -fPIC -larpack -fopenmp -c ./op.f90
+gfortran -fPIC -larpack -fopenmp -c ./diag.f90
+gfortran -fPIC -larpack -fopenmp -c ./diag_re.f90
+gfortran -fPIC -shared -larpack -fopenmp -L ${libdir} -o $prefix/libfuzzified.$dlext ./*.o
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libfuzzified", :LibpathFuzzifiED)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="LLVMOpenMP_jll", uuid="1d63c593-3942-5779-bab2-d838dc0a180e"))
```

Comment:
```suggestion
    # For OpenMP we use libomp from `LLVMOpenMP_jll` where we use LLVM as compiler (BSD
    # systems), and libgomp from `CompilerSupportLibraries_jll` everywhere else.
    Dependency("CompilerSupportLibraries_jll"; platforms=filter(!Sys.isbsd, platforms)),
    Dependency("LLVMOpenMP_jll"; platforms=filter(Sys.isbsd, platforms)),
```

For the non-BSD systems, we need the libgomp library.